uefi-test-runner/src/main.rs

```diff
@@ -10,13 +10,18 @@ extern crate log;
 #[macro_use]
 extern crate alloc;
 
+extern crate x86_64;
```

Comment:
Since we're using Rust 2018 we can remove this and simply directly use the crate through the prelude